cove_iati/lib/iati.py

```diff
@@ -447,3 +479,93 @@ def organisation_identifier_count(tree):
     unique_identifiers = list(set(identifiers))
 
     return len(unique_identifiers)
+
+
+ACTIVITY_PREFIX = '/iati-activities/iati-activity'
+
+
+def check_activity_org_refs(tree):
+    root = tree.getroot()
+    
+    publishers = requests.get("https://codelists.codeforiati.org/api/json/en/ReportingOrganisation.json").json()
+    registration_agency = requests.get("https://codelists.codeforiati.org/api/json/en/OrganisationRegistrationAgency.json").json()
```

Comment:
As discussed earlier, it would be good to deal with these JSON files being offline, in a way that doesn't break the whole results page.